### ISSUE TITLE
[A11y]Manage Organisation_Reflow: On zooming the screen content to 400% both the scrollbar appear to view the screen.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8489]
Link: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1294821

## Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org
Operating System: Windows 10
Browser Version: Chrome Version 85.0.4183.121 (Official Build) (64-bit),
Chromium Edge Version 87.0.637.0 (Official build) dev (64-bit)

## Repro Steps:​
 Hit the URL " https://www.nuget.org" and login with appropriate credentials.
Tab Navigate to "Sign in" control and hit enter.
 Tab Navigate to "Sign in with Microsoft" control and hit enter.
Select “Manage Organizations” from the Profile Dropdown and hit enter.
Tab navigate to browser setting and zoom the screen content at400% and observe horizontal/vertical scroll bar present to view the content.
## Actual:​
When the users zoom the screen content to 400% the data on the screen can be viewed with both the scrollbar i.e. Vertical and Horizontal which is incorrect as just one of the scrollbar show be available.
## Expected:​
The data on the screen should be viewed with only one of the scrollbar when zoomed to 400%.
## ​User Impact:​
When users zoom the screen content to 400% and if they have to view the screen content with both the scrollbar then they will find difficulty in viewing and reading the content present on the screen.

### Notes
There was a similar bug filed a while back that was fixed by moving the overflow on the to table, since scrollbar on page is the issue, not scrollbar on particular content.
